tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrerTest.xtend

```diff
@@ -0,0 +1,162 @@
+package org.testeditor.tcl.dsl.jvmmodel
+
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.dsl.common.testing.DummyFixture
+
+class TclJvmModelInferrerTest extends AbstractTclGeneratorIntegrationTest {
+
+	@Before
+	def void parseAmlModel() {
+		parseAml(DummyFixture.amlModel + '''
+			component type dummyComponentType {
+				interactions = stop
+			}
+			component dummyComponent is dummyComponentType {
+				element dummyElement is Label {
+					locator = "dummyLocator"
+				}
+			}
+		''').assertNoSyntaxErrors
+	}
+
+	@Test
+	def void testGenerationForResolvedFixture() {
+		val tclModel = parseTcl('''
+			package com.example
+			
+			# MyTest
+			
+			* test something
+			
+			Component: dummyComponent
+			- Stop application
+		''')
+		tclModel.addToResourceSet
+		
+
+		val tclModelCode = tclModel.generate
+
+		tclModelCode.assertEquals('''
```

Comment:
as we talked about, an assertMatches() should be enough. There is no need to explicitly check for string equality on all the surrounding code ...